backend/data/road_segmenting/calculate_distance.py

```diff
@@ -0,0 +1,74 @@
+from math import *
+import utm
+
+
+def distance_formula(start_list, end_list):
+    """
+    Calculates the distance between two points using the distance formula
+    :param start_list: the first GPS point, given in UTM list form [start_x, start_y]
+    :param end_list:  the second GPS point, given in UTM list form [start_x, start_y]
+    :return: The distance between the first GPS point and the second GPS point
+    """
+    distance = sqrt(pow((start_list[0] - end_list[0]), 2) + pow((start_list[1] - end_list[1]), 2))
+    return distance
+
+
+def harvesine_formula(start_list, end_list):
```

Comment:
Do a project-wide rename here, the name is [Haversine](https://en.wikipedia.org/wiki/Haversine_formula), not Harvesine.